identity/src/commonMain/kotlin/com/android/identity/securearea/SecureArea.kt

```diff
@@ -135,10 +135,11 @@ interface SecureArea {
     /**
      * Gets information about a key.
      *
+     * This works even on keys that are invalidated.
```

Comment:
Invalidated keys could be removed or replaced at any time, so usage of this function may appear flaky. Why do we need to carve out this exception in the API? It seems like it'll result in inconsistent behavior that could lead to bugs.

Reply:
We need it b/c most applications will want to build a model of Documents / Credentials (just like we do in DocumentModel in samples/testapp - which probably should be moved to -appsupport) and to do you need to call getKeyInfo() so this is helpful if it keeps working even if keys are invalidate.

I don't understand what you mean by "flaky"... a key being invalidated just means that the private key material is no longer available, all the metadata about the key should still be around.

Reply:
Ah, I was mixing up the Expiration concept in our new Storage API with this key invalidation.
Ok, this makes sense. Thanks.